condition/condition_plugins/src/accelaration_condition.cpp

```diff
@@ -56,7 +56,8 @@ bool AccelerationCondition::update(
 
       if (not (*api_ptr_).getNPCAccel(trigger_, &npc_acceleration))
       {
-        ROS_ERROR_STREAM("Invalid trigger name specified for " << getType() << " condition named " << getName());
+        // TODO(yunus.caliskan): use the logger of the simulator instead.
```

Comment:
Do you want to change this now or is this left for later? What simulator do you mean? I wouldn't know what to fix here if I stumbled over this comment in two months

Reply:
This was waiting on `scenario_api` to be concretized so i can interact with its API. Now that it is in, i fixed it to use the logger of the scenario api.